build.go

```diff
@@ -0,0 +1,355 @@
+package composer
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
```

Comment:
nit: same as before, can we get rid of all usage of `io/ioutil`?

Reply:
Thanks for the catch, I will do this. https://pkg.go.dev/io/ioutil